bin/desi_submit_prod

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import argparse
+
+import sys
+from desispec.scripts.submit_prod import submit_production
+
+def parse_args():  # options=None):
+    """
+    Creates an arguments parser for the desi run production
+    """
+    parser = argparse.ArgumentParser(description="Submit a full production run of the DESI data pipeline for processing.")
+
+    parser.add_argument("-p", "--production-yaml", type=str, required=True,
+                        help="Relative or absolute pathname to the yaml file summarizing the production.")
+    parser.add_argument("-q", "--queue-threshold", type=int, default=4800,
```

Comment:
Let's make this 4500 to give a little bit more overhead before the 5000 hard limit.